src/test-helpers/typeFactory.ts

```diff
@@ -81,6 +81,34 @@ export class TypeFactory {
 		return key.setMeta(storageEntry.creator.meta);
 	}
 
+	/**
+	 * Similar to the storageKey method above, this is specific to storageKeys
+	 * that take in double map for keys.
+	 *
+	 * @param index The id to assign the key to.
+	 * @param indexTypeOne The First InterfaceType that will be used to create the index into its new appropriate index type
+	 * @param indexTypeOne The Second InterfaceType that will be used to create the index into its new appropriate index type
+	 * @param storageEntry Used primarily on QueryableStorageEntry (ie: api.query) within the polkadot api library.
+	 * Contains necessary key value pairs to retrieve specific information from a query
+	 * such as `at`, `entriesAt`, `entries` etc..
+	 *
+	 * Some Parameter Examples:
+	 * 1. apiPromise.query.assets.account
+	 */
+	storageKeyDoubleMap(
+		index: number,
+		indexTypeOne: keyof InterfaceTypes,
+		indexTypeTwo: keyof InterfaceTypes,
+		storageEntry: StorageEntryBase<'promise', GenericStorageEntryFunction>
+	): StorageKey {
+		const idOne = this.#registry.createType(indexTypeOne, index);
```

Comment:
The here is wrong, I had just named it because the doubleMap i was directly building this for was `AssetId`, and `AccountId` so i naturally wrote `id` switching it to `mapValueOne` and `mapValueTwo`